src/containers/Ledger/index.js

```diff
@@ -37,6 +37,57 @@ const DATE_OPTIONS = {
   timeZone: TIME_ZONE,
 }
 
+const TransactionCategories = {
+  NFT: 'NFT',
+  DEX: 'DEX',
+  PSEUDO: 'PSEUDO',
+  XCHAIN: 'XCHAIN',
+  ACCOUNT: 'ACCOUNT',
+  PAYMENTS: 'PAYMENTS',
+}
+
+const mapTransactionTypesToCategory = {
+  NFTokenBurn: TransactionCategories.NFT,
+  NFTokenMint: TransactionCategories.NFT,
+  NFTokenAcceptOffer: TransactionCategories.NFT,
+  NFTokenCancelOffer: TransactionCategories.NFT,
+  NFTokenCreateOffer: TransactionCategories.NFT,
+
+  OfferCancel: TransactionCategories.DEX,
+  OfferCreate: TransactionCategories.DEX,
+
+  CheckCash: TransactionCategories.PAYMENTS,
+  CheckCancel: TransactionCategories.PAYMENTS,
+  CheckCreate: TransactionCategories.PAYMENTS,
+  EscrowCancel: TransactionCategories.PAYMENTS,
+  EscrowCreate: TransactionCategories.PAYMENTS,
+  EscrowFinish: TransactionCategories.PAYMENTS,
+
+  Payment: TransactionCategories.PAYMENTS,
+  PaymentChannelFund: TransactionCategories.PAYMENTS,
+  PaymentChannelClaim: TransactionCategories.PAYMENTS,
+  PaymentChannelCreate: TransactionCategories.PAYMENTS,
+
+  TrustSet: TransactionCategories.ACCOUNT,
+  AccountSet: TransactionCategories.ACCOUNT,
+  AccountDelete: TransactionCategories.ACCOUNT,
+  SetRegularKey: TransactionCategories.ACCOUNT,
+  SignerListSet: TransactionCategories.ACCOUNT,
+  DepositPreauth: TransactionCategories.ACCOUNT,
+
+  SetFee: TransactionCategories.PSEUDO,
+  UNLModify: TransactionCategories.PSEUDO,
+  EnableAmendment: TransactionCategories.PSEUDO,
+
+  XChainClaim: TransactionCategories.XCHAIN,
+  XChainCommit: TransactionCategories.XCHAIN,
+  XChainModifyBridge: TransactionCategories.XCHAIN,
+  XChainCreateBridge: TransactionCategories.XCHAIN,
+  XChainCreateClaimID: TransactionCategories.XCHAIN,
+  XChainAddAttestation: TransactionCategories.XCHAIN,
+  SidechainXChainAccountCreate: TransactionCategories.XCHAIN,
```

Comment:
```suggestion
  XChainAccountCreateCommit: TransactionCategories.XCHAIN,
```

Reply:
Is there any way to set up this map so it's easier to update when new transactions are added? Given that this is on the `Ledgers` page, it's easy to forget to add a transaction to it when you add a new transaction to the Explorer.

Reply:
Perhaps it can be added to the `TransactionMapping` object for new transactions.